runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpAutoPauseLogoutBehavior.java

```diff
@@ -0,0 +1,20 @@
+package net.runelite.client.plugins.xptracker;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum XpAutoPauseLogoutBehavior
+{
+	NEVER("Never"),
+	ON_LOGOUT("On Logout"),
+	AFTER_LOGOUT_1("Logout + 1m"),
```

Comment:
Is this option really useful?

Reply:
It seemed like a compromise for discussions on issues and in discord I think.